### ISSUE TITLE
Store seed phrase in icloud keychain

### ISSUE THREAD

**Issue by satindergrewal** [Meshbits/pirate-chain-ios-wallet#12]
Storing seed phrase in icloud keychain has cons and pros:

#### Cons:
In any case like history of iCloud account(s) hacking goes, the risk is the user's seed phrase gets compromised via icloud account hacked, and that very well possibly may result in compromising and losing funds in pirate chain address associated to that seed phrase.

#### Pros:
The user gets convenient option to backup and restore their seed phrase related to the application.

--------

So, understanding both pros and cons, I propose to make option of storing the seed phrase in iCloud keychain only after user acknowledges the risk that in any case their iCloud account gets compromised, they can potentially lose their funds as well.

In that case under settings **make "Backup wallet" option show another screen**, instead of straight showing the wallet seed phrase. The options under "Settings -> Backup wallet" should look like this:

- Write seed phrase to paper _[need to write process of it in another issue]_
- Print paper wallet _[will link related issue here later]_
- Backup wallet seed to iCloud

On selecting option "Backup wallet seed to iCloud" show user a warning:

```
Storing the seed phrase in iCloud will provide convenience to store backup of my wallet
which can be used to restore the wallet on another device in case current device is lost or
unusable to access pirate chain wallet funds.

Checkbox: I acknowledge that in case my iCloud account gets compromised, my pirate chain
wallet funds can potentially gets hacked as well.
```

Under that, give 2 buttons:
- Understood risk, please proceed
- No thanks

Only after the "Understood risk, please proceed" option selected take backup to iCloud account.
